source/Userdocs/Software/PyLEMS.md

```diff
@@ -1 +1,49 @@
+(pylems)=
 # PyLEMS
+
+An application written in Python which can serve as an API, as well as a simulator for
+the [LEMS](http://lems.github.io/LEMS) language, and which can also be used to run NeuroML2
+models (by including the LEMS core type definitions).
+
+
+## Quick start
+
+```
+pip install pylems        # install using pip
+pylems -h                 # get info on usage
+pylems lemsexample.xml    # load and simulate a LEMS file
+
+```
+
+Note: to simulate a NeuroML file you must specify the location of the [NeuroML 2 LEMS definitions](https://github.com/NeuroML/NeuroML2/tree/master/NeuroML2CoreTypes) with the -I option, e.g.
+```
+pylems -I <dir of NeuroML2 install>/NeuroML2CoreTypes/  LEMS_NeuroML2_Model.xml
+```
+
+or alternatively simply use {ref}`pyNeuroML <pyNeuroML>`, which bundles these files.
+
+## Source
+
+PyLEMS is under development on GitHub at [https://github.com/LEMS/pylems](https://github.com/LEMS/pylems).
+
+## Documentation
+
+More information can be found [here](https://github.com/LEMS/pylems).
```

Comment:
We should link to the docs, not the repo. We don't want users to look at the code unless they want to report bugs/contribute?

http://lems.github.io/LEMS/